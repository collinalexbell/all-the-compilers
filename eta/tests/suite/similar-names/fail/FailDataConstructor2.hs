
module Main where

data Hello = HelloWorld deriving ( Show )
data World = Helloworld deriving ( Show )

main = do
  print HelloWorld
  print Helloworld

