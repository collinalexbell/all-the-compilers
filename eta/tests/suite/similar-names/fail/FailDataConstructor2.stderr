
tests/suite/similar-names/fail/FailDataConstructor2.hs:5:14: error:
    Multiple declarations with names differing only in case.
    Declared at: tests/suite/similar-names/fail/FailDataConstructor2.hs:4:14
                 tests/suite/similar-names/fail/FailDataConstructor2.hs:5:14
